Diagnostics/AVTester/Test-ExchAVExclusions.ps1

```diff
@@ -391,6 +394,7 @@ $ModuleAllowList.add("ExDbFailureItemApi.dll")
 $ModuleAllowList.add("Microsoft.Cloud.InstrumentationFramework.Metrics.ni.dll")
 $ModuleAllowList.add("IfxMetrics.dll")
 $ModuleAllowList.add("ManagedBlingSigned.dll")
+$ModuleAllowList.add("ManagedBlingSigned.ni.dll")
```

Comment:
This list should be adjusted to have the default `name.dll` files, then at the end, we also need to add in `name.ni.dll` to avoid this type of adding occurring right now we have a mixed results which we don't want.

Reply:
Or, we could have this added in as the test.

```
        # Clear out modules from the allow list
        foreach ($module in $ModuleAllowList) {
            $ProcessModules = $ProcessModules | Where-Object { $_.ModuleName -ne $module -and $_.ModuleName -ne $($module.Replace(".dll", ".ni.dll")) }
        }
```

But, in order to do that we need to have all the `.dll`s to be lower case.